data_collection/web/guides.rst

```diff
@@ -9,4 +9,224 @@ Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem
+
+```
+// add other image formats
+_paq.push(['setDownloadExtensions', "svg|xcf"]);
+```
+
+Or remove some of extenstions from the existing list with `removeDownloadExtensions`:
+
+```
+_paq.push(['removeDownloadExtensions', "jpg|jpeg"]);
+_paq.push(['trackPageView']);
+```
+
+Tracking downloads beyond file extensions
+"""""""""""""""""""""""""""""""""""""""""
+
+If your case of download links does not fall in above cases you still have options to use, to tell tracker that link should be tracked as a download.
+
+You can add a download attribute to a link HTML tag. eg. 
+
+```<a href="/target-file" download>```
+
+Or if you have to be strict with your HTML, you can add a HTML tag class. Default classes are `piwik_download` and `piwik-download`. Eg. 
+
+```<a href="/taget-file" class="piwik-download">```
+
+Additionally you can define your custom CSS classes for download links with ours :ref:`Javascript Tracker API<force-tracking-download-using-css-class>`. Eg. 
+
+```
+_paq.push(['setDownloadClasses', "custom-download-class"]);
+_paq.push(['trackPageView']);
+```
+
+or you can define a list of classes at once, by passing an array list of CSS classes:
+
+```
+_paq.push(['setDownloadClasses', ["custom-download-class", "other-download-class", "another-class"]]);
+_paq.push(['trackPageView']);
+```
+
+and in HTML code:
+
+```<a href="/taget-file" class="custom-download-class">```
+
+.. note::
+You have to remember that using `setDownloadClasses` always overwrite current list of CSS classes.
+
+
+Tracking downloads with inline Javascript
+"""""""""""""""""""""""""""""""""""""""""
+
+There is another alternative for above methods. You can track a download with inline javascript. Insert inline javascript to HTML tag with onclick attribute:
+
+```
+<a href="https://piwik.pro/document-url" target="_blank" onClick="_paq.push(['trackLink', 'https://piwik.pro/document-url', 'download']);">Download document</a>
+```
+
+Tracking downloads when using log importer
+""""""""""""""""""""""""""""""""""""""""""
+
+When you use the :ref:`Log Importer<data-collection-web-log-analytics>`, files with one of the file extensions listed above will be automatically tracked as downloads in Piwik PRO.
+
+Outlinks
+^^^^^^^^
+The Piwik PRO `Outlinks` report shows the list of external URLs that were clicked by your visitors. Outlinks are links that have different domain than those configured for your website. For example, if your visitor click on a link to `piwik.pro` and your website domain is `example.org`, this will be reported as an outlink, no matter if the website opens in current tab/window or a new one. For each outlink, Piwik PRO will report the number of clicks, and the number of unique clicks. Unique clicks are the number of unique visitors that clicked to a given domain. If one visitor clicks multiple times on one outlink it will be reported as one unique click and multiple clicks. For example `http://piwik.pro/` will have 5 clicks and 1 unique click. 
```

Comment:
I'm not sure if this:
> Piwik PRO will report the number of clicks, and the number of unique clicks. Unique clicks are the number of unique visitors that clicked to a given domain. If one visitor clicks multiple times on one outlink it will be reported as one unique click and multiple clicks. For example `http://piwik.pro/` will have 5 clicks and 1 unique click.

is needed here. It is the reporting part

Reply:
Removed

> For each outlink, Piwik PRO will report the number of clicks, and the number of unique clicks. Unique clicks are the number of unique visitors that clicked to a given domain. If one visitor clicks multiple times on one outlink it will be reported as one unique click and multiple clicks. For example `http://piwik.pro/` will have 5 clicks and 1 unique click.

part